sysl2/sysl/cmd_runner.go

```diff
@@ -105,3 +121,73 @@ func EnsureFlagsNonEmpty(cmd *kingpin.CmdClause, excludes ...string) {
 
 	cmd.PreAction(fn)
 }
+
+func (r *cmdRunner) getProjectRoot(fs afero.Fs, logger *logrus.Logger) error {
+	rootIsDefined := r.Root != ""
+
+	modulePath := r.module
+	if rootIsDefined {
+		modulePath = filepath.Join(r.Root, r.module)
+	}
+
+	syslRootPath, err := findRootFromSyslModule(modulePath, fs)
+	if err != nil {
+		return err
+	}
+
+	rootMarkerExists := syslRootPath != ""
+
+	switch {
+	case rootIsDefined:
+		if rootMarkerExists {
+			logger.Warningf(rootMarkerExistsWarning, syslRootMarker, syslRootPath, r.Root)
+		} else {
+			logger.Warningf(noRootMarkerWarning, syslRootMarker, r.Root)
+		}
+	case !rootIsDefined && !rootMarkerExists:
```

Comment:
PLEASE be consistent with your codee style. You used a if inside the case above but this uses extra bool e
expressions. Should have just made it:
```
if rootIsDefined {
   if roootMarkerExists { } else {}
} else {
   if roootMarkerExists { } else {}
}

```

Reply:
fixed